engine/api/project.go

```diff
@@ -157,9 +156,29 @@ func addProjectHandler(w http.ResponseWriter, r *http.Request, db *gorp.DbMap, c
 		return sdk.WrapError(err, "AddProject> Cannot insert project")
 	}
 
+	var groupAttached bool
+
+	for i := range p.ProjectGroups {
+		groupPermission := &p.ProjectGroups[i]
+		if strings.TrimSpace(groupPermission.Group.Name) == "" {
+			continue
+		}
+		groupAttached = true
+	}
+	if !groupAttached {
+		permG := sdk.GroupPermission{
+			Group:      sdk.Group{Name: p.Name + sdk.DefaultGroupOnProjectSuffix},
```

Comment:
i'm not sure, i am happy with this.
If you want to make project permission easier to manager in CDS, creating a group for each project won't solve this problem. 
BTW i admit that we have something to do about this

Reply:
The group is automatically created only if user don't select an existing group. But yes, it's only a suggestion

Reply:
In fact, there two kinds of users:
- newcomers: they use the same group name as project name. Today, they often think that the group is only usable on this project (group name like "admins" for example)
- users familiar with CDS: they probably want to choose an existing group, so they can still choose the group.

Reply:
yup, the permissions are way too complicated to newcomers.
They should be able to work on their first pipeline after 2 clicks on UI